config/ci_configs/amip_coarse_ft64_hourly_checkpoints_restart.yml

```
@@ -3,8 +3,7 @@ dt_save_restart: "10days"
 dt_save_to_sol: "1days"
 energy_check: false
 h_elem: 6
-hourly_checkpoint: true
-hourly_checkpoint_dt: 1
+checkpoint_dt: 1
```

does this need to be:
```suggestion
checkpoint_dt: "1hours"
```
?

---

Yes, I missed it!